gas-oracle/flags/flags.go

```diff
@@ -131,10 +131,10 @@ var (
 		EnvVar: "GAS_PRICE_ORACLE_SIGNIFICANT_FACTOR",
 	}
 	PriceBackendURL = cli.StringFlag{
-		Name:     "PriceBackendURL",
-		Usage:    "price exchange backend url",
-		EnvVar:   "PRICE_BACKEND_URL",
-		Required: true,
+		Name:   "PriceBackendURL",
+		Value:  "https://api.bybit.com",
```

Comment:
Don't give the default value, it doesn't feel good

Reply:
OK, modified.